ci/deploy-rpm.sh

```diff
@@ -2,6 +2,26 @@
 
 TRIVY_VERSION=$(find dist/ -type f -name "*64bit.rpm" -printf "%f\n" | head -n1 | sed -nre 's/^[^0-9]*(([0-9]+\.)*[0-9]+).*/\1/p')
 
+function create_common_rpm_repo () {
+        rpm_path=$1
+
+        ARCHES=("x86_64" "aarch64")
+        for arch in ${ARCHES[@]}; do
+                prefix=$arch
+                if [ "$arch" == "x86_64" ]; then
+                        prefix="64bit"
+                fi
+                if [ "$arch" == "aarch64" ]; then
```

Comment:
Hello, thank you for the contribution, seems to me that you could make use of the `elif` bash statement.

```suggestion
                elif [ "$arch" == "aarch64" ]; then
```

Reply:
@realFranco nice catch! thanks.